### DISCUSSION TITLE
Restore Saves to Original Cartridge

### DISCUSSION THREAD

**Discussion by RJDoran93** [sanni/cartreader#191] [General Chat]
Hello. I'm very interested in building one of these cart readers and backing up my entire collection. My only issue is that I can't find information on weather or not this device can restore save game data to a cartridge. I'm hoping to use this device to back up my data so I can replace the save batteries on my games and then put the data back. Is this possible? Thank you for your time.

**Comment by sanni** [comment]
Yes, generally speaking restoring save games works.

**Reply by RJDoran93** [reply]
Awesome! Sorry if that question was asked before. I tried searching the discussion area, but I'm new to GitHub and have no idea what I'm doing. Is there a manual or something I can read to learn how to put the data back and all the things the software can do? Thank you again.

**Comment by sanni** [comment]
There is a basic manual here: https://github.com/sanni/cartreader/wiki
And a feature list here: https://github.com/sanni/cartreader#supported-systems

**Reply by RJDoran93** [reply]
Thank you so much Sanni! You're the best!
